protocol/superchainerc20/L1-native-factory.md

```diff
@@ -0,0 +1,170 @@
+## Summary
+
```

Comment:
Can we call the implementation of the converted `SuperchainERC20` tokens `OptimismSuperchainERC20` tokens? So that the naming is consistent between `OptimismMintableERC20` and `OptimismSuperchainERC20`, its the optimism mintable erc20 that can be converted into optimism superchain erc20

Then we can keep `SuperchainERC20` as the name of the standard

Reply:
I will update the naming in Summary and Overview as well